src/components/UserInfo.js

```diff
@@ -1,18 +1,23 @@
 export default class UserInfo {
-  constructor() {
-    this._profileName = document.querySelector(".profile__title");
-    this._profileJob = document.querySelector(".profile__description");
+  constructor({ profileName, profileJob, profileAvatar }) {
+    this._profileName = document.querySelector(profileName);
+    this._profileJob = document.querySelector(profileJob);
+    this._profileAvatar = document.querySelector(profileAvatar);
   }
 
   getUserInfo() {
     return {
       name: this._profileName.textContent,
-      job: this._profileJob.textContent,
+      about: this._profileJob.textContent,
     };
   }
 
-  setUserInfo({ title, description }) {
-    this._profileName.textContent = title;
-    this._profileJob.textContent = description;
+  setUserInfo({ name, about }) {
+    this._profileName.textContent = name;
+    this._profileJob.textContent = about;
+  }
+
+  setUserAvatar({ avatar }) {
```

Comment:
It's great to have a separate method for updating the avatar as it's controlled by a different form